app/__init__.py

```diff
@@ -12,8 +13,7 @@ def create_app(model_path=None):
     if model_path is None:
```

Comment:
Could you make an environment variable out of this? Then we'll be able to configure containers

Reply:
I added a config file with the path to the default model set